src/content/docs/logs/logs-context/get-started-logs-context.mdx

```diff
@@ -161,14 +165,29 @@ For more on these options, see the agent-specific logs-in-context docs.
 
 ## Log metrics [#log-metrics]
 
-The logging metrics reported by <InlinePopover type="apm" /> agents are displayed on the <DoNotTranslate>**Logs**</DoNotTranslate> chart on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page. These metrics are separate from the [log forwarding feature](#apm-logs-defined) and cannot be disabled at the account level. To disable logging metrics, see the dedicated APM configuration docs (for example, [this `logging_metrics` config option for PHP](/docs/apm/agents/php-agent/configuration/php-agent-configuration/#cfg-application_logging_metrics-enabled)).
+When you set up an <InlinePopover type="apm" /> agent, you automatically get a **Logs** chart of logging metrics on the APM <DoNotTranslate>**Summary**</DoNotTranslate> page: 
+
+<img
+    style={{ align: 'left',maxWidth: '70%' }}
+    title="Screenshot showing the logging metrics chart"
+    alt="Screenshot showing the logging metrics chart"
+    src={logsLoggingMetrics}
+/>
+
+This chart shows a count of logs, and if you haven't disabled APM log forwarding, you can click on the chart links that will take you to the logs themselves. Even if you turn off APM log forwarding, this chart still shows the potential logs you could inspect if you had APM log forwarding enabled. 
```

Comment:
I think there should be a comma between the "and" and the "if", which makes me think maybe the sentence should be tweaked or broken out to not have so many commas. Really small nitpick though.

Reply:
@jeff-colucci yes, I missed the comma for that parenthetical phrase that is conditional. Still, I think your point is that  this is a confusing sentence. I've changed it to this:

```
This chart shows a count of logs. If you haven't disabled APM log forwarding, you can click on the chart links that will take you to the logs themselves. 

```